lib/commands/element.js

```diff
@@ -103,57 +102,33 @@ commands.getSize = async function getSize (el) {
   return {width: rect.width, height: rect.height};
 };
 
-function hasSpecialKeys (keys) {
-  for (let char of keys) {
-    if (isSpecialKey(char)) {
-      return true;
+function prepareInputValue (inp) {
+  if (!_.isArray(inp) && !_.isString(inp)) {
+    throw new Error(`Only strings and arrays are supported as input arguments. ` +
+      `Received: ${JSON.stringify(inp)}`);
+  }
+
+  // possible values of `inp`:
+  //   ['some text']
+  //   ['s', 'o', 'm', 'e', ' ', 't', 'e', 'x', 't']
+  //   'some text'
+  // make it into an array of single characters
```

Comment:
This is not making an array, but making a string from an array, if necessary. Or is this comment meant to be for the rest of the function? In which case I think an empty line would be useful, to indicate so.

Reply:
ok, will add an empty line ;)